### ISSUE TITLE
unblocked a blocked model

### ISSUE THREAD

**Issue by esgomezm** [bioimage-io/collection-bioimage-io#594]
Hey (@FynnBe ?)

I accidentally let this entire model as blocked, but it should have been only the version. Then I submitted a new model but the pull request didn't open. Now I corrected, will the pull request be generated or do I need to upload the model again?

https://github.com/bioimage-io/collection-bioimage-io/blob/b57ba46d862a53d1ca947c88f9d22f975fa67706/collection/10.5281/zenodo.7781877/resource.yaml#L13

**Comment by FynnBe** [comment]
haha, yeah I thought so... https://github.com/bioimage-io/collection-bioimage-io/pull/589#discussion_r1154156275
no, your correction is fine. as it is no new upload needed. 
new PR: https://github.com/bioimage-io/collection-bioimage-io/pull/595
